packages/open-next/src/core/routing/util.ts

```diff
@@ -105,14 +105,11 @@ export function convertRes(res: OpenNextNodeResponse): InternalResult {
   const isBase64Encoded =
     isBinaryContentType(headers["content-type"]) ||
     !!headers["content-encoding"];
-  // We cannot convert the OpenNextNodeResponse to a ReadableStream directly
-  // You can look in the `aws-lambda.ts` file for some context
-  const body = Readable.toWeb(Readable.from(res.getBody()));
   return {
     type: "core",
     statusCode,
     headers,
-    body,
+    body: Readable.from(res._chunks),
```

Comment:
Some explanation here is worthwhile: the current implementation of `res.getBody()` does a `Buffer.concat(res._chunks)`, which means the entire buffer is copied into memory, then passed into a Node.js `stream.Readable` that gets further wrapped in a `ReadableStream` adapter implementation... when it could just be `ReadableStream.from(res._chunks)`

Btw, @anonrig ... should this be `ReadableStream.from(res._chunks)`? `Readable.from(...)` will return a node `stream.Readable` which means this is getting a different type than the original.

Reply:
You're right but I'm seeing an error with the proposed change.

```
tests-unit:test: TypeError: Cannot read properties of undefined (reading 'Symbol(Symbol.asyncIterator)')
tests-unit:test:  ❯ Module.convertRes packages/open-next/src/core/routing/util.ts:112:26
tests-unit:test:     110|     statusCode,
tests-unit:test:     111|     headers,
tests-unit:test:     112|     body: ReadableStream.from(res._chunks),
tests-unit:test:        |                          ^
tests-unit:test:     113|     isBase64Encoded,
tests-unit:test:     114|   };
tests-unit:test:  ❯ packages/tests-unit/tests/core/routing/util.test.ts:242:20
```

Reply:
Ok, then it likely needs to be wrapped in an async generator...

```js
async function* gen() {
  for (const chunk of res._chunks) {
    yield chunk;
  }
}

ReadableStream.from(gen());
```